lib/router/router.js

```diff
@@ -109,6 +126,15 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && this.options.ifNoOperation && this.options.ifNoOperation.justForward === true) {
```

Comment:
Please use `noop` query. This is consistent with the existing `useOriginal` query.

https://github.com/asilvas/node-image-steam/blob/master/lib/http/connect.js#L138